Fix the same typo in 8 Java exception messages

jbampton opened the pull request:
Update codespell.txt remove `excetion`

### Description

This PR fixes a simple typo in the exception messages.

Hoping simple fixes can be merged quickly.  Let us move faster :rocket: 

<!--- Describe your changes in DETAIL - And how has behaviour functionally changed. -->

<!-- For new features, provide link to FS, dev ML discussion etc. -->
<!-- In case of bug fix, the expected and actual behaviours, steps to reproduce. -->

<!-- When "Fixes: #<id>" is specified, the issue/PR will automatically be closed when this PR gets merged -->
<!-- For addressing multiple issues/PRs, use multiple "Fixes: #<id>" -->
<!-- Fixes: # -->

<!--- ******************************************************************************* -->
<!--- NOTE: AUTOMATION USES THE DESCRIPTIONS TO SET LABELS AND PRODUCE DOCUMENTATION. -->
<!--- PLEASE PUT AN 'X' in only **ONE** box -->
<!--- ******************************************************************************* -->

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [X] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI
- [ ] test (unit or integration test code)

### Feature/Enhancement Scale or Bug Severity

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### Screenshots (if appropriate):

### How Has This Been Tested?

<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, and the tests you ran to -->

#### How did you try to break this feature and the system with this change?

<!-- see how your change affects other areas of the code, etc. -->

<!-- Please read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/main/CONTRIBUTING.md) document -->

sureshanaparti commented:
@blueorangutan package

blueorangutan commented:
Packaging result [SF]: ✔️ el8 ✔️ el9 ✔️ debian ✔️ suse15. SL-JID 14350

DaanHoogland commented on server/src/main/java/com/cloud/vm/snapshot/VMSnapshotManagerImpl.java:
```diff
@@ -539,7 +539,7 @@ public VMSnapshot createVMSnapshot(Long vmId, Long vmSnapshotId, Boolean quiesce
                 throw new RuntimeException("Operation is interrupted", e);
             } catch (java.util.concurrent.ExecutionException e) {
-                throw new RuntimeException("Execution excetion", e);
+                throw new RuntimeException("Execution exception", e);
```
```suggestion
                throw new CloudRuntimeException("Execution exception getting the outcome of the asynchronous create VM snapshot job", e);
```

DaanHoogland commented on server/src/main/java/com/cloud/vm/snapshot/VMSnapshotManagerImpl.java:
```diff
@@ -774,7 +774,7 @@ public UserVm revertToSnapshot(Long vmSnapshotId) throws InsufficientCapacityExc
                 throw new RuntimeException("Operation is interrupted", e);
             } catch (java.util.concurrent.ExecutionException e) {
-                throw new RuntimeException("Execution excetion", e);
+                throw new RuntimeException("Execution exception", e);
```
```suggestion
                throw new CloudRuntimeException("Execution exception getting the outcome of the asynchronous revert to snapshot job", e);
```

DaanHoogland reviewed (commented):
I like the change, but would like to take the chance to also change
1. the use of the generic `RuntimeException` and replace it with our own
2. replace the repetitive message to messages specific to the jobs

weizhouapache commented on .github/linters/codespell.txt:
```diff
@@ -190,7 +190,6 @@ erro
 everthing
 everytime
-excetion
 excption
```
and remove this ?

weizhouapache replied:
Actually I think more words can be removed, if they are not found in code

JoaoJandre commented:
> @weizhouapache @JoaoJandre @sureshanaparti (and others) what do you think;
> 
> > I like the change, but would like to take the chance to also change
> > 
> > 1. the use of the generic `RuntimeException` and replace it with our own
> > 2. replace the repetitive message to messages specific to the jobs

I'm +1 with your suggestion @DaanHoogland

jbampton commented:
> @jbampton , if you are alright with the suggested changes, we can apply those and test??

Done now @DaanHoogland thanks

jbampton commented:
> @weizhouapache I agree with [#11285 (comment)](https://github.com/apache/cloudstack/pull/11285#discussion_r2262809271) but would like to leave it out of scope here…?

Fixed now @DaanHoogland and @weizhouapache I have removed the misspelled words from our ignored words list `codespell.txt`.  So we won't need to fix this again as pre-commit will check and enforce it.

DaanHoogland commented:
@blueorangutan package

DaanHoogland commented:
did an upgrade test
